pipeline/stepBuilder.go

```diff
@@ -26,17 +26,15 @@ import (
 
 	backend_types "go.woodpecker-ci.org/woodpecker/v2/pipeline/backend/types"
 	"go.woodpecker-ci.org/woodpecker/v2/pipeline/errors"
-	pipeline_errors "go.woodpecker-ci.org/woodpecker/v2/pipeline/errors"
```

Comment:
could we use pipeline_errors as there is the std package `errors` witch can be used here in the future?

Reply:
Done.